### PR TITLE
WIP: ENG-1476: Improve MyProfile page

### PR DESCRIPTION
**Pull request by gReis89**
**Blocked by [ENG-1371](https://jira.entando.org/browse/ENG-1371)**

Adds the ability to upload a profile picture.

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/user-profile/selectors.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
```
Comment (ichalagashvili) [inline src/state/user-profile/selectors.js:1]: we can delete this line

File: src/state/user-profile/types.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line
```
Comment (ichalagashvili) [inline src/state/user-profile/types.js:1]: also this

File: src/ui/users/common/DropzoneUploader.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import Dropzone from 'react-dropzone';
+
+const DropzoneUploader = ({ onUpload, maxFiles, accept }) => (
+  <Dropzone onDrop={onUpload} maxFiles={maxFiles} accept={accept} className="DropzoneUploader">
+    {({ getRootProps, getInputProps }) => (
+      <section>
+        <div className="DropzoneUploader__wrapper" {...getRootProps()}>
+          <input {...getInputProps()} />
+          <i className="fa fa-cloud-upload DropzoneUploader__icon" />
+          <p className="DropzoneUploader__text">Drag and drop or <span>Browse your computer</span></p>
```
Comment (ichalagashvili) [inline src/ui/users/common/DropzoneUploader.js:12]: should not this be i18n text?

**Comment by gReis89** [comment]
Closing this one as this feature got implemented on a different ticket after API refactor: https://github.com/entando/app-builder/pull/1044

**Pull request closed without merging**